modules/builds-configuring-post-commit-build-hooks.adoc

```diff
@@ -10,42 +10,48 @@ There are different ways to configure the post build hook. All forms in the foll
 
 .Procedure
 
-* Shell script:
+* Do one of the following options:
 +
+|===
+|Option |Description
+
+|Shell script
+a|
 [source,yaml]
 ----
 postCommit:
   script: "bundle exec rake test --verbose"
 ----
```

Comment:
Good catch! I actually didn't change the yaml at all for this, I just tried to improve the logical construction of the steps (https://docs.openshift.com/container-platform/4.15/cicd/builds/triggering-builds-build-hooks.html#builds-configuring-post-commit-build-hooks_triggering-builds-build-hooks). 

The entire "Builds using BuildConfig" book contains many yaml examples that do not conform to the OpenShift doc guidelines. I'd suggest opening a JIRA so that the writer assigned to this area can address them properly.